### PR TITLE
Fixes #885 save modules to project

### LINKED ISSUE #885: Save Modules with Project

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: src/MoBi.UI/MoBi.UI.csproj [hunk 1]
```diff
@@ -47,7 +47,25 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Accessibility">
```
Comment (msevestre) [inline src/MoBi.UI/MoBi.UI.csproj:50]: Why do we need those DLLs?

Comment (msevestre) [reply src/MoBi.UI/MoBi.UI.csproj:50]: If we do, we need to create a new package for our devepxress nuget stuff

File: src/MoBi.UI/MoBi.UI.csproj [hunk 2]
```diff
@@ -47,7 +47,25 @@
...
+      <HintPath>..\..\..\..\..\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Data.Linq\v4.0_4.0.0.0__b77a5c561934e089\System.Data.Linq.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Deployment">
+      <HintPath>..\..\..\..\..\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Deployment\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Deployment.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Runtime.Serialization.Formatters.Soap">
+      <HintPath>..\..\..\..\..\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Runtime.Serialization.Formatters.Soap\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Runtime.Serialization.Formatters.Soap.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Security">
+      <HintPath>..\..\..\..\..\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Security\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Security.dll</HintPath>
```
Comment (msevestre) [inline src/MoBi.UI/MoBi.UI.csproj:67]: This is all very weird. Why are those DLLs added

Comment (rwmcintosh) [reply src/MoBi.UI/MoBi.UI.csproj:67]: oh geez. I did not look carefully at the csproj. I'll back that out.

**Review by msevestre — CHANGES_REQUESTED** [review]
I am worried about all those dependencies added. Can you explain why we need those coming from GAC and not from the framework? Feels like other deps are linked differently

**Comment by rwmcintosh** [comment]
I don't know how it got added, but I didn't check the csproj files before committing (or even after). This is carelessness.

**Comment by rwmcintosh** [comment]
Fixed

**Pull request merged**